test/parallel/test-repl-preview.js

```diff
@@ -88,6 +88,14 @@ async function tests(options) {
      '\x1B[36m[Function: koo]\x1B[39m',
      '\x1B[1G\x1B[0Jrepl > \x1B[8G'],
     ['a', [1, 2], undefined],
+    ["{ b: 1 }['b'] === 1", [2, 6], '\x1B[33mtrue\x1B[39m',
```

Comment:
Maybe add a whitespace at the beginning and also add another test for `"{ b: 1 }['b'] === 1;"`?

Reply:
so - test the same logic twice, just once with whitespace at start and one without?

Reply:
Yes. It should not have the same result. It should be `false`.